testing/trino-benchto-benchmarks/src/test/resources/hive_metadata/README.md

```diff
@@ -0,0 +1,7 @@
+These metadata files were generated from the glue connector from TPCDS/TPCH schema with sf1000.
+So, there are no min/max statistics for char-based columns.
+
+### How to generate?
+1. Execute `gimme-aws-creds --mfa-code <code>` in your cmdline to fetch
```

Comment:
I think this is specific to our setup. I would rephase it:
```
1. You need to be logged into your AWS account. For example if you use MFA authentication, you can use `gimm...`
```

This also makes us the only ones capable of providing the stats.
Could you maybe add short, general description how one can put TPCH/TPCDS sf1000 data on Glue?

Reply:
> This also makes us the only ones capable of providing the stats.

The hive catalog used by HiveMetadataRecorder can be configured to point to anything, it's not special to our setup.

Reply:
added a better doc